build.gradle

```diff
@@ -45,7 +45,7 @@ apply plugin: 'com.palantir.failure-reports'
 
 javaVersions {
     libraryTarget = 11
-    runtime = 21
+    runtime = 17
```

Comment:
Surprised to find this in this PR?

Reply:
This was temp bc I was running into errors with java 21 downloading the jdk. Turns out vpn was blocking some cert during installation to gradle-jdks